spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/SparkBigQueryConfig.java

```diff
@@ -300,12 +300,12 @@ public static SparkBigQueryConfig from(
         date -> validateDateFormat(date, config.getPartitionTypeOrDefault(), DATE_PARTITION_PARAM));
     // checking for query
     if (tableParam.isPresent()) {
-      String tableParamStr = tableParam.get().trim().replaceAll("\\s+", " ");
-      if (isQuery(tableParamStr)) {
```

Comment:
Is there any reason to remove the tableParamStr variable? You can just remove the replaceAll call